### DISCUSSION TITLE
备份和恢复问题，备份到本地却没有备份进文件小提琴，只有在备份管理里面能看到有备份，备份管理里面的备份不能进行任何操作，历史无用的备份不能删除，而恢复备份却要从文件系统里面找备份，关键是文件系统里面又没有备份，整个备份和恢复的逻辑是一团糟

### DISCUSSION THREAD

**Discussion by juno2580** [xiaohucode/yidaRule#130] [General]
备份和恢复问题，备份到本地却没有备份进文件小提琴，只有在备份管理里面能看到有备份，备份管理里面的备份不能进行任何操作，历史无用的备份不能删除，而恢复备份却要从文件系统里面找备份，关键是文件系统里面又没有备份，整个备份和恢复的逻辑是一团糟

**Comment by xiaohucode** [comment]
## 在备份管理可以删除或恢复
![QQ截图20240323212509](https://github.com/xiaohucode/yidaRule/assets/24434389/364355c9-ad86-4cdb-abf9-bbef62a9bd13)

**Reply by juno2580** [reply]
苹果手机版不行

**Reply by juno2580** [reply]
![2430C47F-D783-4E95-BAA2-572139980C76](https://github.com/xiaohucode/yidaRule/assets/131781194/5e8cbc78-87ff-481e-8297-fcad2da8c905)

**Reply by xiaohucode** [reply]
菜单按钮没适配夜间模式，关闭夜间模式就能看到了

**Comment by juno2580** [comment]
> 苹果手机版不行
